### PR TITLE
File lease

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-file-share/tests/test_file.py [hunk 1]
```diff
@@ -848,6 +1034,33 @@ def test_copy_file_with_existing_file(self):
         copy_file = file_client.download_file().readall()
         self.assertEqual(copy_file, self.short_byte_data)
 
+    # TODO:uncomment this in STG71 copy file PR
```
Comment (xiafu-msft) [inline sdk/storage/azure-storage-file-share/tests/test_file.py]: I will address this after copy file pr is merged

File: sdk/storage/azure-storage-file-share/tests/test_file.py [hunk 2]
```diff
@@ -968,6 +1181,32 @@ def test_unicode_get_file_unicode_name(self):
         # Assert
         self.assertEqual(content, b'hello world')
 
+    @record
+    def test_unicode_get_file_unicode_name_with_lease(self):
+        # Arrange
+        file_name = '啊齄丂狛狜'
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-file-share/tests/test_file.py]: >啊齄丂狛狜 [](start = 21, length = 5)

:joy:

File: sdk/storage/azure-storage-file-share/tests/test_file.py (hunk 1 above)
Comment (xiafu-msft) [reply sdk/storage/azure-storage-file-share/tests/test_file.py]: addressed

**Pull request merged**